### ISSUE TITLE
Closing game client in combat.

### ISSUE THREAD

**Issue by higuysitme** [FeenixServerProject/Phoenix_1.12.1_Issue_tracker#97]
You can avoid death by closing game client during combat(alt+f4) in instances/dungeons. If nps do not have you in target you will survive when you log back in. It is major abuse allowing avoid complete wipe of raid. Moreover, you can alt f4 before nefarian lands if you do not have Onyxia Scale Cloak so you will not get hit by initial shadowflame damage. You supposed to die if you alt+f4 in combat.

**Comment by Ifixstuffk** [comment]
True, any disconnect during combat in a raid resulted in death.

**Comment by Kaiitothewai** [comment]
No, that's not true at all. You did not die while in combat if you disconnected (This also applies to ALTF4'ing). 

https://youtu.be/vU6WQk9eLJI?t=2m30s

However, what you are both looking for is that characters that disconnect in any way should not be removed from the game server, before a certain amount of time has passed. Also to note, I am quite positive that if you were combat, you should not be logged out till you either died or dropped combat naturally. The characters ingame should act just like a normal character would, however they would stop doing auto attacks upon disconnecting, but they should be taking damage and be able to get healed, just like any other online character would.

This would obviously mean that if a character that disconnects on the Nefarian landing that does not have the cloak equipped, would still die as they cannot be removed from the game server as they're in combat. They'd get hit by the dot along with the initial hit due to not them wearing the Onyxia Scale Cloak and thus dieing, as intended.

**Comment by higuysitme** [comment]
Ye its blizzlike mechanics to alt f4 if you about to wipe. Seems legit. Lold.

**Comment by Kaiitothewai** [comment]
Honestly, I have no idea if Feenix has implemented ANTI-ALTF4 features / disconnect mechanics, but there's a difference between creating a ALTF4 Disconnect killer and a legitmate blizzlike fix.

I don't quite grasp the funny part of having such a major bug infest the raiding scene yet to this day, but each to their own.
